Add Repository Sets as a new procedure (#2126)
theforeman/foreman-documentation: guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

> [id="Changing_the_Repository_Sets_Status_in_Project_{context}"]

Reviewer: ```suggestion
[id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
```

= Changing the Repository Sets Status in {Project}

> Repository sets show available repositories to each host. If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.

Reviewer: ```suggestion
Repository sets show available repositories to each host.
If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```

> If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.

Reviewer 2: No need to repeat this here (it's more relevant below.) But instead it might be good to expand a bit on the overview of what Repository sets is used for.

```suggestion
Repository sets show repositories available to each host. A host will be able to access content from a given repository if that repository is enabled.
```

.Procedure

> . In the {ProjectWebUI}, navigate to *Hosts* to choose from a list of available hosts.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
```

> . Select the *Repository sets* tab.

Reviewer 2: ```suggestion
. Select the *Content* tab, then the *Repository sets* subtab.
```

. Select the *Repository sets* tab.
On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.

> . You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.

Reviewer 2: You can also bulk-select with the checkboxes on each table row and use the action menu at the top.

> . Navigate to the *Overview* tab to *Content view details* to view the environment for the host.

Reviewer: ```suggestion
. On the *Overview* tab, click *Content view details* to view the environment for the host.
```

. On the *Overview* tab, click *Content view details* to view the environment for the host.
+
[NOTE]
====

> For hosts not in a default status or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.

Reviewer 2: Re-commenting here so this doesn't get lost in the resolved comment. "Default status" isn't correct -- we use the word "default" here in two different ways. Really the meaning here is "for hosts not in the Default Organization View content view and Library lifecycle environment," but that's a bit cumbersome. We should also highlight that this is what makes the toggle group show. Maybe something like this:

```suggestion
For hosts not in the default content view and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, 'Limit to Environment' and 'Show all.'

Limit to environment: Show only repositories that are relevant to the host.
Show all: Show all available repositories, even those that may not be in the host's content view and lifecycle environment.
```
```suggestion
For hosts not in a default status or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.
```

> For hosts not in a default view or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.

Reviewer: What do you mean by "default view"?

Author: Changing it to `default status` which is set when you choose `Reset to default`.

Reviewer 2: Here "default view" means "the Default Organization View content view and Library lifecycle environment" which is a bit cumbersome. We should also highlight that this is what makes the toggle group show. Maybe something like this:

```suggestion
For hosts not in the default content view and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, 'Limit to Environment' and 'Show all.'

Limit to environment: Show only repositories that are relevant to the host.
Show all: Show all available repositories, even those that may not be in the host's content view and lifecycle environment.
```

> +

Reviewer: ```suggestion
```

[id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]

> = Changing the Repository Sets Status in {Project}

Reviewer 2: Since this procedure applies to hosts (and not AKs or bulk hosts), should this info be added to the title?
```suggestion
= Changing the Repository Sets Status for a Host in {Project}
```

Repository sets show repositories available to each host.
A host will be able to access content from a given repository if that repository is enabled.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
. Select the *Content* tab, then select the *Repository sets* subtab.
On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.

> . You can also bulk select the checkboxes on each table row and use the action menu at the top.

Reviewer 2: I didn't mean for my exact words to be here.. if there's a better or more standardized way to say this, please update! I'm sure there are lots of procedures that document using the vertical ellipsis / kebab / action menu thingie at the top of the table, but I wasn't sure how that's usually handled.

Author: Totally meant to change that but pushed the commit without thinking. Oops!

. On the *Overview* tab, click *Content view details* to view the environment for the host.
+
[NOTE]
====
For hosts not in the default Content View and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, *Limit to environment* and *Show all*.

Limit to environment: Show only repositories that are relevant to the host.

> Show all: Show all available repositories including those that may not be in the host's Content View and lifecycle environment.

Reviewer 2: I'm not that familiar with ASCIIdoc formatting, but I feel like this might need some. Otherwise it will just be a regular prose paragraph, right?
